src/ray/object_manager/transfer_queue.h

```diff
@@ -61,11 +61,13 @@ class TransferQueue {
   ///
   /// \param client_id The ClientID to which the object needs to be sent.
   /// \param object_id The ObjectID of the object to be sent.
-  void QueueSend(const ClientID &client_id, const ObjectID &object_id,
+  void QueueSend(const ClientID &client_id, const ObjectID &object_id, uint64_t data_size,
```

Comment:
Update the documentation for the params here and in other places in the file.

Reply:
These have been updated.